include/bluetooth/rpc.rst

```diff
@@ -0,0 +1,72 @@
+.. _ble_rpc:
+
+Bluetooth Low Energy Remote Procedure call
+##########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF Connect SDK has support for the Bluetooth Low Energy stack serialization.
+The full Bluetooth Low Energy stack can be ran on the another device or CPU in particular on the nRF5340 DK Network Core using :ref:`nrfxlib:nrf_rpc`.
+
+.. note::
+   Currently a serialization of the :ref:`zephyr:bt_gap` and :ref:`zephyr:bluetooth_connection_mgmt` is supported.
+
+Network Core
+************
+
+The :ref:`ble_rpc_host` sample is designed specifically to enable the Bluetooth LE stack functionality on a remote MCU (for example, the nRF5340 network core) using the :ref:`nrfxlib:nrf_rpc`.
+This sample can be programmed to the network core to run standard Bluetooth Low Energy samples on nRF5340.
+You can choose whether to use the SoftDevice Controller or the Zephyr Bluetooth LE Controller for this sample.
+
+Application Core
+****************
+
+Bluetooth sample which are running on Application sample must be built with additional configuration.
+When building samples for the Application core, enable the :option:`CONFIG_BT_RPC` to run Bluetooth Low Energy stack on the Network Core.
+This option also causes that the :ref:`ble_rpc_host` is built automatically as a child image.
+For more details see: :ref:`ug_nrf5340_building`.
+
+.. tabs::
+
+   .. group-tab:: west
+
+      Open a command prompt in the build folder of the application sample and enter the following command to build application for the Application Core and :ref:`ble_rpc_host` as child image::
+
+        west build -b nrf5340dk_nrf5340_cpuapp -- -DCONFIG_BT_RPC=y
+
+Requirements
+************
+
+Some configuration options related to the Bluetooth Low Energy must be equal on the host (Network core) and client (Application core).
```

Comment:
Should we inform the reader that all those settings that must be equal on both sides can be placed in a dedicated Kconfig fragment and then shared ?

So users can do:
```
$ west build -b <board> -- -DCONFIG_OVERLAY=bt_rpc.conf
```

That will make it much easier to keep those settings in sync.

Reply:
Good point, I can add something like that at the end of file:

```
A convenient way of keeping all the above configuration options in sync is to create an overlay file shared between the Application and the Network core. Then, you can invoke build command like this:

.. code-block:: console

   west build -b <board> -- -DCONFIG_OVERLAY=my_overlay_file.conf
```

Reply:
... between the application core and network core. (no capitalization)

Then, instead of `code block` use:
.. parsed-literal::
   :class: highlight

   west build -b *board* -- -DCONFIG_OVERLAY=my_overlay_file.conf